Revision of section 4.3 to improve its clarity (#637)
cf-convention/cf-conventions: ch04.adoc

[[vertical-coordinate, Section 4.3, "Vertical Coordinate"]]
=== Vertical (Height or Depth) Coordinate

Variables representing vertical coordinates must have either

> * a **`positive`** attribute with a value of **`up`** or **`down`** (case insensitive), which respectively mean that increasingly positive coordinate values indicate higher locations i.e. further from the centre of the Earth (**`positive="up"`**) or lower locations (**`positive="down"`**); or

Reviewer: ```suggestion
the attribute **`positive`** that takes either the value **`up`** or **`down`** (case insensitive), which respectively mean that increasingly positive coordinate 
```
